### DISCUSSION TITLE
Beeing able to use channel emotes in the tab names when renaming them

### DISCUSSION THREAD

**Discussion by JustusShG** [Chatterino/chatterino2#2836] [Ideas] [answered]
Currently you can only use emojis and normal text in the tab names. Would it be possible to use the channel emotes in the respective channel for the tab?

For example, if I rename a tab and put a channel emote in it, would it be possible to display the emote?

**Comment by pajlada** [comment, marked as answer]
Technically, yes, it would be possible to implement but the amount of work required is immense and a can of worms which would inevitably lead to ugly code.
